stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/attachment/AttachmentSelectionListener.kt

```diff
@@ -0,0 +1,19 @@
+package io.getstream.chat.android.ui.message.input.attachment
+
+import com.getstream.sdk.chat.model.AttachmentMetaData
+
```

Comment:
How about OnAttachmentsSelectedListener / onAttachmentsSelected ?

Reply:
Why would you change the listener name to that? Looking at our current listeners, we have very few that have the `On` prefix (I'm seeing 4 in total).

Reply:
You are right